src/components/App.js

```diff
@@ -1,77 +0,0 @@
-import React, { Component } from 'react';
-import {connect} from 'react-redux'
-import {bindActionCreators} from 'redux'
-
-import {checkLocation, echoWebSocket, testLatency, getWeather} from '../actions/main'
-
-import WeatherCanvas from './WeatherCanvas'
-
-import css from '../style/main.less'
```

Comment:
Partial URLs are interpreted relative to the source of the style sheet, not relative to the document